### ISSUE TITLE
question about the open vocabulary performance of Grounding DINO on the COCO dataset

### ISSUE THREAD

**Issue by ymzis69** [jianzongwu/Awesome-Open-Vocabulary#27]
Thank you for your amazing work. I currently have a question regarding the open vocabulary performance of Grounding DINO on the COCO dataset and hope to get your response.

At present, I am researching open vocabulary performance. In your paper, other models achieve a maximum AP<sub>all</sub> of only 61.0 on the COCO dataset when the IoU threshold is 0.5. However, MM Grounding DINO (reproduction of Grounding DINO under the MMDetection framework, pretrained on O365, GoldG, GRIT, and V3Det datasets, available at https://github.com/open-mmlab/mmdetection/blob/main/configs/mm_grounding_dino/README.md) achieves 73.6 for this metric on the COCO dataset. Could you please confirm if these are the same test metrics? If they are the same, does this mean that the open vocabulary performance of MM Grounding DINO significantly surpasses other methods? I look forward to your reply.
![1718101457449](https://github.com/jianzongwu/Awesome-Open-Vocabulary/assets/67489888/5663eb66-a09d-4e5c-b603-db0606005881)
![1718101480131](https://github.com/jianzongwu/Awesome-Open-Vocabulary/assets/67489888/aa1e60de-5b5e-4db8-8075-3115821f60e9)

**Comment by lxtGH** [comment]
@ymzis69 Hello, as one co-author of MM-Grounding DINO, this is because MM-Grounding DINO use more datasets than original Grounding DINO. Please cite the original number when performing comparison.

**Comment by ymzis69** [comment]
Thank you for your response, but Grounding DINO did not train on the base classes of the COCO dataset and then test the results on both base and novel classes, which is why I referenced the results from MM-Grounding DINO. To be more precise, I am not investigating the model's zero-shot performance on the COCO dataset, but rather the model's performance on novel classes after being trained on the base classes of the COCO dataset.

**Comment by lxtGH** [comment]
For me, I think it is hard to directly compare Grounding DINO results with Tab-9 in our work since most objects are seen during the pre-training.
